src/main/java/com/google/cloud/discotoproto3converter/proto3/Message.java

```diff
@@ -38,6 +59,49 @@ public Message(String name, boolean ref, boolean isEnum, String description) {
     return fields;
   }
 
+  public Map<Integer, Field> getFieldsWithNumbers() {
+    Map<Integer, Field> fieldsWithNumbers = new LinkedHashMap<>();
+
+    List<Field> sortedFields =
+        fields.stream()
+            .skip(isEnum ? 1 : 0)
+            .sorted(Comparator.comparing(Field::getName))
+            .collect(Collectors.toList());
+    if (isEnum) {
+      sortedFields.add(0, fields.get(0));
+    }
+
+    for (Field f : sortedFields) {
+      if (fieldsWithNumbers.isEmpty() && isEnum()) {
+        fieldsWithNumbers.put(0, f);
```

Comment:
OK, so looking at the baselines, I see that the zeroth enum value, which is always our dummy-unset value, always gets the value zero. Could you add a comment to that effect?

Reply:
Added the comment